docs/IPC_MESSAGES.md

```diff
@@ -190,7 +190,28 @@ Response:
     }
 }
 ```
-
+### `UpdateNewContractOnDeployment` message
+Request:
+```
+{
+    id : <unique_request_id>,
+    type : UpdateNewContractOnDeployment,
+    address : ...,
+    bytecode : [Secret Contract Address]
+    delta : [First delta]
+}
+```
+Response:
+```
+{
+    id : <unique_request_id>,
+    type : UpdateNewContractOnDeployment,
+    address : ...,
+    result : {
+        status : 0 or err code
```

Comment:
please add to `result` an extra field: `errors: [contract_status, delta_status]` so we will know when an error occurred, where exactly it happened.